src/components/category-box/category-box-styles.module.scss

```diff
@@ -0,0 +1,45 @@
+.box {
+  background-color: rgba(9, 20, 26, 0.45);
+  box-shadow: 0 2px 10px 0 rgba(23, 27, 48, 0.15);
+  padding: 35px 30px 30px 20px;
+  width: 280px;
+  margin-bottom: 5px;
+  transition: all 0.2s ease-out;
+
+  &:hover {
+    transform: translate(20px, 0);
+  }
+}
+
+.visible {
+  cursor: pointer;
+}
+
+.title {
+  color: rgba(white, 0.6);
+  font-size: 10px;
+  font-family: 'PierSans';
```

Comment:
Hey @aabdaab maybe we can create some `settings.scss` file to store all this variables related to fonts and colours.